todo_app/data/todo_app_items.py

```diff
@@ -0,0 +1,48 @@
+import pymongo
+import datetime
+import os
+from todo_app.data.data_item import Data_Item
+
+def connection_mongo():
+
+    client = pymongo.MongoClient(os.getenv("CONNECTION_STRING"))
+    
+    db = client.jp_todoapp
+
+    posts = db.posts
```

Comment:
Minor, but all the use of `posts` in this file suggests we've let the tutorial collection name persist into our code without thinking too hard; actually it's arguably a fair name, but worth noting that this could be anything at all, e.g. `collection = db.todo_collection` or similar (although if you were to now switch to use a new name, you would lose the items that are stored in your `posts` collection!)

Reply:
Ah. Thanks Jack.

I thought we had to call it Posts lol 
But I now take it that 'Posts' is just a tutorial example? Not a keyword in the code?

![image](https://github.com/jeanpaulvanr/Devops-Course-Starter/assets/110428360/cf59b269-4ce3-4bdb-b48c-6c464853d09f)

so I could call Posts in the above screenshot something like ToDo_App_Tasks (as that's what they are really).

reflected in the code as 
todo_app_tasks = db.todo_app_tasks

But I'd of course have to go and make the collection: todo_app_tasks in Azure first right?

And I get that that means the existing ones would disappear (as they would exist in a different collection).